pyomo/contrib/incidence_analysis/common/tests/test_dulmage_mendelsohn.py

```diff
@@ -0,0 +1,185 @@
+#  ___________________________________________________________________________
+#
+#  Pyomo: Python Optimization Modeling Objects
+#  Copyright 2017 National Technology and Engineering Solutions of Sandia, LLC
+#  Under the terms of Contract DE-NA0003525 with National Technology and 
+#  Engineering Solutions of Sandia, LLC, the U.S. Government retains certain 
+#  rights in this software.
+#  This software is distributed under the 3-clause BSD License.
+#  ___________________________________________________________________________
+
+import pyomo.common.unittest as unittest
+from pyomo.common.dependencies import (
+        networkx as nx,
+        networkx_available,
+        )
+from pyomo.contrib.incidence_analysis.common.dulmage_mendelsohn import (
+        dulmage_mendelsohn,
+        )
```

Comment:
PEP8: ou editor appears to be indenting these by 8 spaces and not 4

Reply:
Seconding this comment.